include/boost/numeric/ublas/matrix.hpp

```diff
@@ -258,6 +258,30 @@ namespace boost { namespace numeric {
             return at_element (i, j);
         }
 
+	/** Access a matrix element. Here we return a reference
+	 * \param i the first coordinate of the element. By default it's the row
+	 * \param j the second coordinate of the element. By default it's the column
+	 * \return a reference to the element
+	 */
+		BOOST_UBLAS_INLINE
```

Comment:
The indentation of this block of code is off. Please adjust to match the rest of the code. (The same comment applies to a few other places.)

Reply:
Will use clang-format now for that purpose.

Reply:
I have not used clang-format. Need to aggree on the format first. So I have used qtcreator for that purpose.